blocks/algorithms/__init__.py

```diff
@@ -276,7 +278,7 @@ def compute_steps(self, gradients):
 
 
 class SteepestDescent(StepRule):
-    """A step in the direction opposite to the gradient.
+    """A step in the direction proportional to the gradient.
```

Comment:
This gets a bit confusing now...

If we want this terminology to make sense than I would argue that all `StepRule`s no longer take `gradient` as an input (because it's no longer receives a gradient, it's either the negative gradient, or something completely different in the case of chaining). The `SteepestDescent` rule would then be `ScaleRule` or something, because that is all it does (scaling the step it receives, the first step being hardcoded to be the negative gradient by the `GradientDescent` class).